lib/schema.ncl

```diff
@@ -24,7 +25,16 @@ let nix = import "./nix-interop/nix.ncl" in
     shells
       | OrganistShells
       | optional,
-    flake | FlakeOutputs | optional,
+    files
+      | filegen.Files
+      | doc m%"
+        Set of files that should be generated in the project's directory.
+      "%
+      | default
+      = {},
+    flake | FlakeOutputs = {
+        apps.regenerate-files.program = nix-s%"%{filegen.regenerate_files files}/bin/regenerate-files"%,
```

Comment:
Should probably also add a check that would verify that files are properly generated here.

Reply:
Indeed. Let me save that for a follow-up PR